modules/bezug_lgessv1/JSON-Beispiele.txt

```diff
@@ -0,0 +1,69 @@
+Login:
+
+$ curl -k --connect-timeout 5 -d '{"password":"DE1807BKRD0093E8"}' -H "Content-Type: application/json" -X PUT https://192.168.0.248/v1/login
```

Comment:
**Attention:**  
Is this your active password that has been published to the world audience now?  
Then remember that it's time to change it!